### DISCUSSION TITLE
Benchmark vs other products

### DISCUSSION THREAD

**Discussion by bioinformatist** [CeresDB/ceresdb#842] [Q&A] [answered]
What's the performance CeresDB compared to e.g. TDengine and TimescaleDB?

**Comment by ShiKaiWi** [comment, marked as answer]
Sorry for late response, we didn't pay enough attention to the discussions here.

Currently, we focus on the improvement of performance, especially, write performance in this month, so there is no plan to do benchmark compared with these timeseries databases you mentioned, which may take much time to deploy them and tune them. However, the benchmark compared with influxdb was performed and recorded when releasing the v1.0, here is the [link](https://github.com/CeresDB/community/blob/main/posts/release-v1.0/index.md#performance-optimization-and-benchmark).

And such benchmark will be conducted again at the end of this month when releasing v1.2, which may deserve your attention.

**Reply by bioinformatist** [reply]
Thanks for reply. And will v1.2 come with K8s supported?

**Reply by ShiKaiWi** [reply]
The first version of the deployment tool for K8S is planned to be released with v1.2 if the tool can catch up the deadline of v1.2.
